src/main/common/filter.c

```diff
@@ -81,8 +81,11 @@ FAST_CODE float slewFilterApply(slewFilter_t *filter, float input)
 
 
 float filterGetNotchQ(uint16_t centerFreq, uint16_t cutoff) {
-    float octaves = log2f((float) centerFreq  / (float) cutoff) * 2;
-    return sqrtf(powf(2, octaves)) / (powf(2, octaves) - 1);
+//    float octaves = log2f((float) centerFreq  / (float) cutoff) * 2;
```

Comment:
the commented code can most likely be removed (I think at least)

Reply:
I  was not sure ... using octaves will match textbook implementation, I left it as documentation. But it may be moved into comment of function ..

Reply:
Understandable 👍. Perhaps we can include a link to the Wolfram calculation as well? Just a thought.